.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "geewallet"]
+	path = geewallet
+	url = https://github.com/aarani/geewallet.git
```

Comment:
@aarani I merged the PR so pls update above

Reply:
Done